server/api.go

```diff
@@ -727,23 +727,31 @@ func (p *Plugin) createIssueComment(w http.ResponseWriter, r *http.Request, user
 		return
 	}
 
+	commentUsername := ""
 	commentUser, appErr := api.GetUser(post.UserId)
 	if appErr != nil {
 		writeAPIError(w, &APIErrorResponse{ID: "", Message: "failed to load post.UserID " + post.UserId + ": not found", StatusCode: http.StatusInternalServerError})
 		return
 	}
+	if info, err := p.getGitHubUserInfo(post.UserId); err != nil {
```

Comment:
While the logic is correct here, only a subset of errors returned by `getGitHubUserInfo` indicate that the user is not connected. I think it's valuable that these get logged. You can check the error type with 
```golang
	if info, apiEr := p.getGitHubUserInfo(post.UserId); apiEr != nil {
		if apiErr.ID == API_ERROR_ID_NOT_CONNECTED` {
```

Reply:
Done !